src/java.base/share/classes/java/time/format/DateTimeFormatterBuilder.java

```diff
@@ -2787,15 +2787,10 @@ public boolean format(DateTimePrintContext context, StringBuilder buf) {
                 }
             } else {
                 switch (signStyle) {
-                    case NORMAL:
-                    case EXCEEDS_PAD:
-                    case ALWAYS:
-                        buf.append(decimalStyle.getNegativeSign());
-                        break;
-                    case NOT_NEGATIVE:
-                        throw new DateTimeException("Field " + field +
-                            " cannot be printed as the value " + value +
-                            " cannot be negative according to the SignStyle");
+                    case NORMAL, EXCEEDS_PAD, ALWAYS -> buf.append(decimalStyle.getNegativeSign());
+                    case NOT_NEGATIVE                -> throw new DateTimeException("Field " + field +
+                                                            " cannot be printed as the value " + value +
+                                                            " cannot be negative according to the SignStyle");
```

Comment:
Maybe in this case you could revert the alignment. In this case it would allow lines 2792 to 2793 to be shorter

Reply:
I've removed the alignment of arrow operators as requested. See 24ae9e5